hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenUpdateHandler.java

```diff
@@ -166,7 +168,7 @@ public void handle(@NonNull final HandleContext context) throws HandleException
         // enabled and has open slots. If so, auto-associate.
```

Comment:
It's probably not the time now, but in the future we should consider baking this comment into the code by giving these separate checks _names_ so that we can just read the names and know what checks are being performed - rather than having to map these various conditions for auto-association onto the confusing mass of code below.